kong/dao/schemas_validation.lua

```diff
@@ -117,9 +117,16 @@ function _M.validate_entity(tbl, schema, options)
               is_valid_type = bool == "true" or bool == "false"
               t[column] = bool == "true"
             elseif v.type == "array" then
-              t[column] = utils.strip(t[column]) == "" and {} or utils.split(t[column], ",") -- Handling empty arrays
+              -- handle escaped commas inside the comma-separated array
+              -- by flipping them into a \0 and then back
+              local escaped = t[column]:gsub("\\,", "\0")
+              if utils.strip(t[column]) == "" then
+                t[column] = {} -- Handling empty arrays
+              else
```

Comment:
yep, can do. from the looks of the code I thought the line-break rule applied to larger blocks, not one-liners